CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changes
+
+## [Unreleased] - ReleaseDate
+
+## [0.1.0] - 2023-10-24
+This is egglog's first release! Egglog is ready for use, but is still fairly experimental. Expect some significant changes in the future.
+
+- Egglog is better than [egg](https://github.com/egraphs-good/egg) in many ways, including performance and new features.
+- Egglog now includes cargo documentation for the language interface.
+
+As of yet, the rust interface is not documented or well supported. We reccomend using the language interface. Egglog also lacks proofs, a feature that egg has.
+
+
+[Unreleased]: https://github.com/egraphs-good/egglog/compare/v0.1.0...HEAD
```

Comment:
What's `Unreleased`?

Reply:
It's the in progress version between v0.1.0 and the next version
That way you get a link to everything that has happened in main since v0.1.0 and people can gradually update the changelog

Reply:
I see. Great!